File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int, int> map;

        for (int i = 0; i < nums.size(); i++) {
            int b = target - nums[i];
            if (map.count(b)) {
                return {map[b], i};
            }
            map[nums[i]] = i;
        }

        return {-1, -1}; 
    }
};

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] output = {-1,-1};
        for(int f=0; f<nums.length-1;f++){
            for(int s = f+1; s<nums.length;s++){
                if(nums[f]+nums[s]==target){
                    output[0]=f;
                    output[1]=s;
                    break;
                }
            }
        }
        return output;
    }
}